test/extended/etcd/etcd_backup_noconfig.go

```diff
@@ -0,0 +1,376 @@
+package etcd
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	configv1alpha1 "github.com/openshift/api/config/v1alpha1"
+	exutil "github.com/openshift/origin/test/extended/util"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/apiserver/pkg/storage/names"
+	"k8s.io/klog/v2"
+	"k8s.io/kube-openapi/pkg/util/sets"
+	"k8s.io/utils/pointer"
+
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+	"github.com/pkg/errors"
+)
+
+const (
+	defaultBackupCRName       = "default"
+	testSchedule              = "*/5 * * * *"
+	testTimeZone              = "UTC"
+	testRetentionType         = configv1alpha1.RetentionTypeNumber
+	testRetentionNumber       = 3
+	backupServerContainerName = "etcd-backup-server"
+	masterNodeLabel           = "node-role.kubernetes.io/master"
+	OpenShiftEtcdNamespace    = "openshift-etcd"
+	backupVolume              = "/var/lib/etcd-auto-backup"
+	backupDirName             = "etcd-auto-backup-dir"
+
+	// ShellImage allows us to have basic shell tooling, taken from origin:
+	// https://github.com/openshift/origin/blob/6ee9dc56a612a4c886d094571832ed47efa2e831/test/extended/util/image/image.go#L129-L141C2
+	ShellImage = "image-registry.openshift-image-registry.svc:5000/openshift/tools:latest"
+)
+
+var _ = g.Describe("[sig-etcd][OCPFeatureGate:AutomatedEtcdBackup][Serial] etcd", func() {
```

Comment:
Can we tag these tests with `[apigroup:config.openshift.io]` so they get skipped in MicroShift?

Reply:
@pacevedom It is featuregated !!

yes I figured this out this few minutes ago, add 

```
"[sig-etcd][OCPFeatureGate:AutomatedEtcdBackup][Serial] etcd is able to apply the no-config backup configuration [Timeout:50m][apigroup:config.openshift.io/v1alpha1]": " [Suite:openshift/conformance/serial]",
```

check https://github.com/openshift/origin/pull/29098/files now please 👍🏽